### ISSUE TITLE
working on chat in hapi-chat

### ISSUE THREAD

**Issue by ZooeyMiller** [dwyl/chat#41]
In regards to https://github.com/dwyl/apprenticeship/issues/8 @finnhodgkin and I are going to work on building a chat in a component style. 

Because we are not familiar enough with Elixir we don't feel comfortable doing this with the backend that is currently in place so we are going to do it in a staging branch on https://github.com/dwyl/hapi-socketio-redis-chat-example so that we can play with the back end (since it is a server we are familiar with). 

We also do not have push access to this repo so doing it in the hapi chat repo makes more sense for now.

Please let us know if you want us to move it or what is preferable.

**Comment by nelsonic** [comment]
@ZooeyMiller please take a look at: https://github.com/dwyl/apprenticeship/issues/10
while I/we catch up on the Chat "backend". thanks. 👍

**Comment by finnhodgkin** [comment]
@nelsonic Sounds good! I've been meaning to learn AWS for a while :+1:

**Comment by nelsonic** [comment]
@finnhodgkin yeah, it's not something any of the cohorts have done so I thought it would be good for you gents to brush up on it. 😉
